Gruntfile.js

```diff
@@ -46,6 +46,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-nodeunit');
 
-  grunt.registerTask('default', ['jshint', 'nodeunit']);
+  grunt.registerTask('test', ['jshint', 'nodeunit']);
```

Comment:
What's wrong with leaving this as the default task? I don't see what value the `test` alias adds.